src/ui/molecules/stripe-elements.svelte

```diff
@@ -141,13 +113,6 @@
 
   $: stripeLocale = getLocaleToUse(initialLocale);
 
-  $: {
-    // @ts-ignore
-    if (unsafeElements && unsafeElements._elements.length > 0) {
-      elements = unsafeElements;
-    }
-  }
```

Comment:
This also looks like a behavioral change, we are no longer checking `unsafeElements._elements.length > 0` before initializing `elements`

Reply:
As discussed in a huddle I'll change this to be a more explicit check of the fact that the elements loaded.

I'll add that check to the confirm and submit functions.

Reply:
Added the paymentElementReadyForSubmission variable to explicitly handle this scenario:

https://github.com/RevenueCat/purchases-js/pull/432/files#diff-05306f59947d840db7ba303affe21d7bbe6b996f0188079c771ef86093498005R47